clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java

```diff
@@ -87,6 +88,11 @@ public ActivateJobsCommandStep3 timeout(final Duration timeout) {
 
   @Override
   public ActivateJobsCommandStep3 workerName(final String workerName) {
+    if (workerName == null) {
+      builder.setWorker(ZeebeClientBuilderImpl.DEFAULT_JOB_WORKER_NAME);
```

Comment:
:x: IMO: using "default" as the worker name is not exactly the expected behavior.

I would expect that the default job worker name from the configuration is used instead of the constant. 

Consider the following scenario:
- I set "my-worker" as the default job worker name in the config
- I pass `null` as the worker name in the builder
- Expected: the worker name is "my-worker" 
- Actual: the worker name is "default"

The behavior could be implemented by adding this null-check in the configuration builder.

Reply:
In this scenario, what will happen if I don't have a configuration, or in the configuration the property is `null` ?

Reply:
In the configuration, the default job worker name is set to "default". 

It should not be possible to set the default job worker name in the configuration to `null`.

Reply:
I think I understand your point, I will update the implementation 👍